app/controllers/catalog_controller.rb

```diff
@@ -221,7 +221,11 @@ class CatalogController < ApplicationController
     config.add_show_field "music_publisher_number", label: "Music Publisher Number", accessor: :music_publisher_number, helper_method: :list
     config.add_show_field "related_records", label: "Related Records", accessor: :related_records, helper_method: :render_related_records_component
     config.add_show_field "rights_information", label: "Rights information", accessor: :rights_information, helper_method: :url_list, component: StaffOnlyComponent
-    config.add_show_field "copyright_info", label: "Copyright", accessor: :copyright_status, helper_method: :render_copyright_component
+    config.add_show_field "copyright_info", label: "Copyright", accessor: :copyright_status, helper_method: :render_copyright_component, if: ->(_controller, _config, document) do
+      value = document.copyright_status
+      # if there is no contextMsg, there is no rights information from the copyright service
+      value.present? && value.info.present? && value.info["contextMsg"].present?
+    end
```

Comment:
Can the 'if' be moved into a separate function, or does that logic need to sit in the config

Reply:
needs to sit in the config because it only take a lambda or a value